src/esp/assets/GenericSemanticMeshData.cpp

```diff
@@ -210,6 +227,15 @@ GenericSemanticMeshData::buildSemanticMeshData(
           // color is found in ssd mapping, so is legal color
           semanticID = ssdColorToIDAndRegionIter->second.first;
           regionID = ssdColorToIDAndRegionIter->second.second;
+
+          // TEMP FOR VERT-BASED OBB CALC
+          // aggregate
+          auto vert = semanticData->cpu_vbo_[vertIdx];
+          vertAggregate[semanticID] += vert;
```

Comment:
I have a fear this can get really imprecise with large vertex counts. Using [Math::Algorithms::kahanSum()](https://doc.magnum.graphics/magnum/namespaceMagnum_1_1Math_1_1Algorithms.html#aa4696c9d956d5c6bb1833a29533be545) could minimize the overall error (I hope it works with vectors).

Reply:
yes, kahan sum! Ty for the suggestion.